src/events/api/events.ts

```diff
@@ -47,6 +54,33 @@ export const getAttendanceEvent = async (id: number): Promise<IAttendanceEvent>
   return attendanceEvent;
 };
 
+export const getEventPayment = async (event_id: number): Promise<IPayment> => {
+  try {
+    const ret = await get<IPayment>(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_PAYMENT_URL}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne ikke hente betalingsinformasjon for arrangementet!');
+  }
+};
+
+export const getEventExtras = async (event_id: number): Promise<IExtra[]> => {
+  try {
+    const ret = await get<IExtra[]>(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_EXTRAS_URL}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne ikke hente valgmuligheter for arrangementet!');
+  }
+};
+
+export const getPublicAttends = async (event_id: number): Promise<IPublicAttedee[]> => {
+  try {
+    const ret: IPublicAttedee[] = await get(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_PUBLIC_ATTENDEES}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne hente påmeldingsliste!');
+  }
+};

```

Comment:
This has already been implemented, see the `publicAttendees` slice and `api/publicAttendee`